lib/process-services-cloud/src/lib/form/components/widgets/data-table/data-table.widget.ts

```diff
@@ -122,6 +124,23 @@ export class DataTableWidgetComponent extends WidgetComponent implements OnInit
         return processVariableDropdownOptions ?? formVariableDropdownOptions;
     }
 
+    private getOptionsFromPath(data: any, path: string): DataRow[] {
```

Comment:
this method https://github.com/Alfresco/alfresco-ng2-components/blob/a8db044092719acdf574c3cffee53308a59b9f2d/lib/process-services-cloud/src/lib/form/components/widgets/dropdown/dropdown-cloud.widget.ts#L128 seems a duplicated and probably we have other places too I wonder if it may is worty to unify it and have once. Please check before if the logic is so related as I think otherwise ignore my comment :)